src/firefly/java/edu/caltech/ipac/firefly/resources/irsa-image-master-table.csv

```diff
@@ -26,6 +26,10 @@ AKARI,AKARI Far-Infrared All-sky Maps,,akari.akari_images,AKARI FIS N60,N60,,FIS
 AKARI,AKARI Far-Infrared All-sky Maps,,akari.akari_images,AKARI FIS N160,N160,,FIS N160 (160 microns),160,160 microns,far-IR,all-sky,all-sky,0.08,2,http://irsa.ipac.caltech.edu/data/AKARI/overview.html,Far-Infrared All-sky Maps N160 (160 microns),ATLAS,file_type='science',image
 AKARI,AKARI Far-Infrared All-sky Maps,,akari.akari_images,AKARI FIS WIDES,WideS,,FIS WideS (90 microns),90,90 microns,far-IR,all-sky,all-sky,0.08,2,http://irsa.ipac.caltech.edu/data/AKARI/overview.html,Far-Infrared All-sky Maps WIDE-S (90 microns),ATLAS,file_type='science',image
 AKARI,AKARI Far-Infrared All-sky Maps,,akari.akari_images,AKARI FIS WIDEL,WideL,,FIS WideL (140 microns),140,140 microns,far-IR,all-sky,all-sky,0.08,2,http://irsa.ipac.caltech.edu/data/AKARI/overview.html,Far-Infrared All-sky Maps WIDE-L (140 microns),ATLAS,file_type='science',image
+MSX,MSX,,msx.msx_images,MSX A (8.28 microns),A,,MSX A (8.28 microns),8.28,8.28 microns,far-IR,all-sky,all-sky,0.08,2,http://irsa.ipac.caltech.edu/data/MSX/overview.html,MSX A,ATLAS,file_type='science',image

```

Comment:
For the title, i'd change it to:  'MSX: Midcourse Space Experiment'

Reply:
Sorry, I completely forgot about it :)

Reply:
Also you should ask Vandana about what is the waveband category (far-ir?) and project type (all-sky?)

Reply:
Done, thanks!